src/dataset/mod.rs

```diff
@@ -81,7 +82,10 @@ impl Float for f64 {
 ///
 /// Labels are countable, comparable and hashable. Currently null-type (no targets),
 /// boolean (binary task) and usize, strings (multi-label tasks) are supported.
-pub trait Label: PartialEq + Eq + Hash + Clone + Ord + fmt::Debug + Default {}
+pub trait Label:
+    PartialEq + Eq + Hash + Clone + Ord + fmt::Debug + Default + Send + Sync + Unpin + Sized
```

Comment:
Also not sure why there are new bounds here.

Reply:
Oh, sorry. Removed them